core/client/queries/get-payment-wallet-with-initialization-data.ts

```diff
@@ -0,0 +1,30 @@
+import { getSessionCustomerAccessToken } from '~/auth';
+import { graphql } from '~/client/graphql';
+
+import { client } from '..';
+
+export async function getPaymentWalletWithInitializationData(entityId: string) {
+  const graphQLQuery = graphql(`
+		query {
+			site {
+				paymentWalletWithInitializationData(filter: {paymentWalletEntityId: "${entityId}"}) {
```

Comment:
This is not the proper way to pass in data in gql.tada queries, check around the codebase to see how to do it properly or check out this example https://github.com/bigcommerce/catalyst/compare/PAYPAL-5114...chore/headless-wallets-cleanup#diff-d5e5c3bd2d9602afaf35db11c6adea51a6a29268df218fe0e76e3035a31d57c5R200-R227.